app/config/locale/translations/en.json

```diff
@@ -40,6 +40,20 @@
     "emails.mfaChallenge.clientInfo": "This verification code was requested using {{b}}{{agentClient}}{{/b}} on {{b}}{{agentDevice}}{{/b}} {{b}}{{agentOs}}{{/b}}. If you didn't request the verification code, you can safely ignore this email.",
     "emails.mfaChallenge.thanks": "Thanks,",
     "emails.mfaChallenge.signature": "{{project}} team",
+    "emails.mfaEnabled.subject": "Two-factor authentication is enabled for your {{project}} account",
+    "emails.mfaEnabled.hello": "Hello {{user}}",
+    "emails.mfaEnabled.body": "Two-factor authentication has been successfully enabled for your {{project}} account.<br/><br/>Please make sure to store your security codes safely, so you can use them if you lose or can’t access your mobile device. Each recovery code can only be used once, but you can re-generate a new set of 6 codes at any time.",
```

Comment:
I'm not sure about adding the recovery code part because those are separate API calls that happen after. Also, if you delete and add, you may not go through the flow of generating the codes again.

Reply:
Split them into it's own file, which should add the bedrock for future console specific translations as well